src/develop/imageop.h

```diff
@@ -378,6 +382,13 @@ typedef struct dt_iop_module_t
   void (*process_tiling)(struct dt_iop_module_t *self, struct dt_dev_pixelpipe_iop_t *piece, void *i, void *o,
                          const struct dt_iop_roi_t *roi_in, const struct dt_iop_roi_t *roi_out,
                          const int bpp);
+  /** NOTE: most likely, you do not want to implement those ^ two functions, but next two: */
+  /** the simplest variant of process(). you can only use OpeMP SIMD here, no intrinsics */
```

Comment:
And this actually shows an issues:
1. What if iop only provides `process_plain()`, and openmp simd codepath is not enabled? should e.g. `default_process()` always fallback to `module->process_plain`, if it is there?